tests/system/python/conftest.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Configuration system/python/conftest.py
+
+"""
+import pytest
+import subprocess
+import os
+
+__author__ = "Vaibhav Singhal"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+
+@pytest.fixture
+def reset_foglamp_start():
+    assert os.environ.get('FOGLAMP_ROOT') is not None
+    subprocess.run(["$FOGLAMP_ROOT/scripts/foglamp kill"], shell=True, check=True)
+    subprocess.run(["echo YES | $FOGLAMP_ROOT/scripts/foglamp reset"], shell=True, check=True)
+    subprocess.run(["$FOGLAMP_ROOT/scripts/foglamp start"], shell=True)
+    stat = subprocess.run(["$FOGLAMP_ROOT/scripts/foglamp status"], shell=True, stdout=subprocess.PIPE)
+    assert "FogLAMP not running." not in stat.stdout.decode("utf-8")
```

Comment:
Can we please assert its running with VERSION variable

Reply:
foglamp start is a shell command with multi line output. Assertion usually fails as we don't get the timely response and hence making the test flaky